lib/pages/getServerSideProps/redirects.js

```diff
@@ -1,22 +1,47 @@
 const getNetlifyFunctionName = require("../../helpers/getNetlifyFunctionName");
 const getDataRouteForRoute = require("../../helpers/getDataRouteForRoute");
+const getLocaleRoutesForI18n = require("../../helpers/getLocaleRoutesForI18n");
 const pages = require("./pages");
 
 const redirects = [];
 
+/** getServerSideProps pages
+ *
+ * Page params {
+ *    route -> '/ssr', '/ssr/[id]'
+ *    filePath -> 'pages/ssr.js', 'pages/ssr/[id].js'
+ * }
+ **/
+
 pages.forEach(({ route, filePath }) => {
   const functionName = getNetlifyFunctionName(filePath);
+  const target = `/.netlify/functions/${functionName}`;
+
+  // Add any locale redirects if i18n
+  getLocaleRoutesForI18n({ route }).forEach((localeRoute) => {
+    redirects.push({
+      route: localeRoute.route,
+      target,
+    });
+    redirects.push({
+      route: localeRoute.dataRoute,
+      target,
+    });
+  });
```

Comment:
I know you created this nice `getLocaleRoutes...` function, but wouldn't it be easier to read/understand like this? 🙃

```suggestion
   const { locales } = require("../../helpers/getI18nConfig")
   
   // ...

  // Add one redirect for every i18n locale
  locales.forEach((locale) => {
    redirects.push({
      route: getRouteWithLocale(route, locale),
      target,
    });
    redirects.push({
      route: getDataRouteWithLocale(dataRoute, locale),
      target,
    });
  });
```